webpack.config.js

```diff
@@ -106,6 +106,15 @@ if ( copyPluginIndex > -1 ) {
 
 module.exports = {
 	...defaultConfig,
+	resolve: {
+		...defaultConfig.resolve,
+		alias: {
+			...defaultConfig.resolve.alias,
+			utils: resolve( './wp-content/themes/core/assets/js/utils' ),
+			common: resolve( './wp-content/themes/core/assets/js/common' ),
+			config: resolve( './wp-content/themes/core/assets/js/config' ),
```

Comment:
wondering if we should add an alias for the theme/blocks directory?

Reply:
Yes, great callout.